### PR TITLE
Add param for map access token

### LINKED ISSUE #53: invalid access token
It would appear that the access token has expired.  Maybe a configurable parameter is the way forward.

https://github.com/mapbox/mbview/blob/97c715f3605869135e0ef9c1d4627462e8db1081/views/map.ejs#L23
screen shot 2016-12-21 at 16 40 31

### PR DESCRIPTION
**Pull request by rodowi**
Closes #53 

Map access token can be set either from the cli or as an environment variable.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var fs = require('fs');
 var utils = require('./utils');
 
 var mbtiles = argv._;
+var accessToken = argv.MapboxAccessToken || process.env.MapboxAccessToken;
```
Comment (snodnipper) [inline cli.js]: In Mapbox GL native we set a MAPBOX_ACCESS_TOKEN environment variable.  Perhaps also include that as discussed in https://github.com/mapbox/mapbox-cli-py/issues/53?

Comment (rodowi) [inline cli.js]: good catch! let me patch this

Comment (rodowi) [inline cli.js]: @snodnipper added patch in e166103

**Comment by snodnipper** [comment]
Works great.  LGTM 👍

**Comment by rodowi** [comment]
rebasing, merging, publishing...

**Comment by rodowi** [comment]
thanks a lot @snodnipper!

**Pull request merged**